specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/Certificates.json

```diff
@@ -0,0 +1,659 @@
+              "description": "CNAME of the certificate to be issued via free certificate",
+              "type": "string"
+            },
+            "domainValidationMethod": {
+              "description": "Method of domain validation for free cert",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CertificateCollection": {
+      "description": "Collection of certificates.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Certificate"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificatePatchResource": {
+      "description": "ARM resource for a certificate.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CertificatePatchResource resource specific properties",
+          "properties": {
+            "friendlyName": {
+              "description": "Friendly name of the certificate.",
+              "type": "string",
+              "readOnly": true
+            },
+            "subjectName": {
+              "description": "Subject name of the certificate.",
+              "type": "string",
+              "readOnly": true
+            },
+            "hostNames": {
+              "description": "Host names the certificate applies to.",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "pfxBlob": {
+              "format": "byte",
+              "description": "Pfx blob.",
+              "type": "string"
+            },
+            "siteName": {
+              "description": "App name.",
+              "type": "string",
+              "readOnly": true
+            },
+            "selfLink": {
+              "description": "Self link.",
+              "type": "string",
+              "readOnly": true
+            },
+            "issuer": {
+              "description": "Certificate issuer.",
+              "type": "string",
+              "readOnly": true
+            },
+            "issueDate": {
+              "format": "date-time",
+              "description": "Certificate issue Date.",
+              "type": "string",
+              "readOnly": true
+            },
+            "expirationDate": {
+              "format": "date-time",
+              "description": "Certificate expiration date.",
+              "type": "string",
+              "readOnly": true
+            },
+            "password": {
```

Comment:
>"password" [](start = 12, length = 10)

Please annotate with x-ms-secret.
[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review)

Reply:
It is still missing x-ms-secret.

---
In reply to: [593534876](https://github.com/Azure/azure-rest-api-specs/pull/13144#discussion_r593534876) [](ancestors = 593534876)